core/cli_entry.py

```diff
@@ -271,3 +270,29 @@ def logging_end_block():
     logging.info("----------------ENDING UPLOAD RUN-----------------")
     logging.info("==================================================")
     logger.remove_directory_logger()
+
+
+class ExitReturn:
```

Comment:
I feel like this should have its own file instead of being in `cli_entry` since it's also used in the GUI.

Reply:
I moved it into it's own file. I added the constants to the top level of the file so the importing will be consistent too.